src/Language/Haskell/GHC/Simple.hs

```diff
@@ -296,7 +295,8 @@ isTargetOf t ms =
 {-# NOINLINE initStaticFlags #-}
 -- | Use lazy evaluation to only call 'parseStaticFlags' once.
 initStaticFlags :: [Located String]
-initStaticFlags = unsafePerformIO $ fmap fst (parseStaticFlags [])
+initStaticFlags = [] -- most probably wrong
```

Comment:
This should be removed. It was really only used to preload static flags back when they existed (only access to it is `seq`), so it doesn't really do anything anymore.